DfE.FindInformationAcademiesTrusts/Pages/Cookies.cshtml.cs

```diff
@@ -72,7 +72,7 @@ private void ApplyCookieConsent()
 
     private void DeleteAppInsightsCookies()
     {
-        var optionalCookiePrefixes = new[] { "ai_session", "ai_user", "ai_authuser" };
+        var optionalCookiePrefixes = new[] { "ai_session", "ai_user", "ai_authUser" };
```

Comment:
While we're renaming this cookie, shall we update it on the cookies table in the UI?